src/routes/docs/advanced/security/gdpr/+page.markdoc

```diff
@@ -1,21 +1,21 @@
 ---
 layout: article
 title: GDPR
-description: The safeguarding of your and your users' data is taken seriously at Appwrite. Learn about Appwrite's measures and compliance with European General Data Protection Regulation (GDPR).
+description: The safeguarding of your and your users' data is taken seriously at Appwrite. Learn about Appwrite's measures and compliance with the European General Data Protection Regulation (GDPR).
 ---
 
-Appwrite compliant with the European General Data Protection Regulation (GDPR). 
-GDPR is a EU regulation that concerns data privacy and security in the European Union and the European Economic Area.
+Appwrite is compliant with the European General Data Protection Regulation (GDPR). 
+GDPR is an EU regulation that concerns data privacy and security in the European Union and the European Economic Area.
 
 By attesting that Appwrite is GDPR compliant, we have done the following:
-- Appwrite users will retain access to their personal information including the right to correct and to delete it.
+- Appwrite users will retain access to their personal information including the right to correct and delete it.
 - Impose the same rules upon the organization’s sub-processors who assist in providing Appwrite’s services as described in the Terms of Service (“ToS”). 
 - Appwrite will notify users promptly about policy changes and/or data breaches.
 
 You can learn more in our [Privacy policy](https://appwrite.io/privacy) and [Cookie policy](https://appwrite.io/cookies). 
 You can also reach us at `privacy@appwrite.io` for more questions.
 
-Appwrite has also implemented the following security measures to achieve technical compliance.
+Appwrite has also implemented the following security measures to achieve technical compliance:
```

Comment:
Please revert this

Reply:
Well, you do it at line 10 of that same file... :)

Reply:
But can remove both if you want